homework/playwright.config.js

```diff
@@ -106,6 +107,8 @@ const config = {
   //   command: 'npm run start',
   //   port: 3000,
   // },
+
```

Comment:
are the additional empty lines needed here?

Reply:
No need, got by accident